src/style.css

```diff
@@ -0,0 +1,141 @@
+/* #region Normalize */
+.page {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+/* #endregion */
+
+/* #region Card */
+.card {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 198px;
+  height: 406px;
+}
+
+.card__logo {
+  margin-top: 32px;
+  margin-left: 19px;
+  margin-right: 19px;
+}
+
+.logo {
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-top: 36px;
+  margin-left: 16px;
+  margin-right: 16px;
+}
+
+.title {
+  color: #060b35;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__info {
+  font-size: 10px;
+  color: #616070;
+}
+
+.info {
+  margin-top: 4px;
+  margin-left: 16px;
+}
+/* #endregion */
+
+/* #region rewiev */
+.review {
+  display: flex;
+  flex-direction: row;
+  margin: 17px 16px 0;
+}
+
+.stars {
+  display: flex;
+  margin-right: 17px;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  border-radius: 0.5px;
+}
+
+.stars--1 > .stars__star:nth-child(-n+1),
+.stars--2 > .stars__star:nth-child(-n+2),
+.stars--3 > .stars__star:nth-child(-n+3),
+.stars--4 > .stars__star:nth-child(-n+4),
+.stars--5 > .stars__star:nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
+
+.review__info {
+  font-size: 10px;
+  display: flex;
+  align-items: flex-end;
+}
+/* #endregion */
+
+/* #region Price */
+.price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-end;
+  gap: 60px;
+  width: 166px;
+  height: 18px;
+  margin: 24px 16px 0;
+}
+
+.price__text {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.price__count {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+/* #endregion */
+
+/* #region Buttom */
+.button {
+  border: 1px solid #00acdc;
+  background: #00acdc;
+  border-radius: 5px;
+  width: 166px;
+  height: 40px;
+  margin: 16px 16px 0;
+  font-weight: 700;
+  font-style: bold;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #fff;
+}
+
+.button:hover {
+  color: #00acdc;
+  background: #fff;
+  border: 1px solid #00acdc;

```

Comment:
don't add the border on hover. look in the checklist

Reply:
all recommendations are guaranteed + created separate blocks for 'card' and 'stars' + added variables.